src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -38,6 +38,7 @@ type Props = {
     isDisabled?: boolean,
     mentionSelectorContacts?: SelectorItems<User>,
     onAnnotationDelete?: ({ id: string, permissions: AnnotationPermission }) => void,
+    onAnnotationEdit?: (id: string, text: string, permissions: AnnotationPermission) => any,
```

Comment:
We should avoid `any` whenever possible.

Reply:
Will use `void` instead.